src/niscope/custom_types/waveform_info.py

```diff
@@ -92,17 +94,17 @@ def __str__(self):
 
         string_representation = ''
         if self.channel is not None:  # We explicitly look for not None to differentiate from empty string
-            string_representation += row_format_s.format('channel', self.channel)
+            string_representation += row_format_s.format('Channel', self.channel)
         if self.record is not None:  # We explicitly look for not None to differentiate from 0
-            string_representation += row_format_d.format('record', self.record)
+            string_representation += row_format_d.format('Record', self.record)
 
         string_representation += row_format_g.format('Absolute X0', self.absolute_initial_x)
         string_representation += row_format_g.format('Relative X0', self.relative_initial_x)
         string_representation += row_format_g.format('dt', self.x_increment)
-        string_representation += row_format_g.format('offset', self.offset)
-        string_representation += row_format_g.format('gain', self.gain)
+        string_representation += row_format_g.format('Offset', self.offset)
+        string_representation += row_format_g.format('Gain', self.gain)
         if self.samples is not None:  # We explicitly look for not None to differentiate from empty array
-            string_representation += row_format_g.format('wfm length', len(self.samples))
+            string_representation += row_format_g.format('Waveform Length', len(self.samples))
```

Comment:
These changes would technically be considered as breaking. But I think it's ok to update them. @marcoskirsch, thoughts?

Reply:
I think it's fine. The same could be said for error codes returned by the runtime changing. I don't expect clients to parse the returned string.